tests/page-objects/reports/reportsPage.wdio.js

```diff
@@ -0,0 +1,17 @@
+const reportListID = '#reports-list';
+const caseId = () => $(`li.indent-0 p span a`);
```

Comment:
I think all of these selectors should include some of the parent selectors, to make sure that we're getting the right elements and that it's obvious to the reader which element we are referring to. Also `indent-0` looks pretty fragile, can we make the selection more robust?